scripts/cita

```diff
@@ -162,11 +162,11 @@ do_start() {
 
     # Start cita-forever
     if [ -z ${debug} ]; then
-        cita-forever -c ${config} start > /dev/null 2>&1
+        cita-forever -c ${config} start
```

Comment:
why do this changes?

Reply:
It wouldn't be logged to console if we don't change this.

Reply:
Maybe just remove `> /dev/null`.
`2>&1` is that error message out to the console.

Reply:
OK.